### PR TITLE
[pyproject] Update packages and python to "^3.6.1"

### PR DESCRIPTION
**Pull request by zhquan**
python = "^3.6.1"
requests = "2.26.0"
urllib3 = "1.26.5"
perceval-weblate = {branch = "master", git = "https://github.com/chaoss/grimoirelab-perceval-weblate"}

poetry.lock updated accordingly.

Signed-off-by: Quan Zhou <quan@bitergia.com>

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -56,6 +56,7 @@ perceval-mozilla = {branch = "master", git = "https://github.com/chaoss/grimoire
 perceval-opnfv = {branch = "master", git = "https://github.com/chaoss/grimoirelab-perceval-opnfv"}
 perceval-puppet = {branch = "master", git = "https://github.com/chaoss/grimoirelab-perceval-puppet"}
 perceval-finos = {branch = "master", git = "https://github.com/Bitergia/grimoirelab-perceval-finos"}
+perceval-weblate = {branch = "master", git = "https://github.com/Bitergia/grimoirelab-perceval-weblate"}
```
Comment (vchrombie) [inline pyproject.toml:59]: I think the repository is moved to the CHAOSS organization. I think we can update the URL.

```suggestion
perceval-weblate = {branch = "master", git = "https://github.com/chaoss/grimoirelab-perceval-weblate"}
```

**Review by vchrombie — CHANGES_REQUESTED** [review]
Thanks, @zhquan.
Looks good but had a small suggestion.

**Review by vchrombie — APPROVED** [review]
LGTM, thanks for working on the changes @zhquan

**Review by sduenas — APPROVED** [review]
LGTM

**Pull request merged**